tests/tests/cross-mount-inode-del.sh

```diff
@@ -0,0 +1,51 @@
+#
+# Test basic correctness of inode deletion behavior across nodes
+#
+
+t_require_mounts 2
+
+sum_counters()
+{
+    local counter="$1"
+
+    cat /sys/fs/scoutfs/*/counters/$counter | awk '{sum += $1} END {print sum}'
+}
+
+echo "== open a file and delete it cross-mount"
+df_2_a=$(sum_counters forest_oino_deletable_false_2)
```

Comment:
If we're going to use counters to isolate our unlink, I think we have to stabilize for previous tests?  We could be able to look at the counters to see that the work execution counter changed without any per-orphan counters changing to see that the orphan work has drained?